device-types/Cisco/UCSC-C220-M6S.yaml

```diff
@@ -0,0 +1,24 @@
+---
+manufacturer: Cisco
+model: UCS-C220-M6S
+slug: cisco-ucs-c220-m6s
+part_number: UCSC-C220-M6S
+u_height: 1
+airflow: front-to-rear
+is_full_depth: true
+console-ports:
+  - name: Console
+    type: rj-45
+interfaces:
+  - name: ens1f0
```

Comment:
These are how the names present in Linux.

IMO, I would name them how they would present in a managed (fabric interconnect connected) environment instead or a generic "Ethernet 1, Ethernet 2"